### PR TITLE
abandon notice

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# This repository is no longer maintained
+
+Due to lack of interest, we had to decide to discontinue this repository. 
+The CMF project will focus on the [Routing component](https://github.com/symfony-cmf/routing) and [RoutingBundle](https://github.com/symfony-cmf/routing-bundle) which is the only part that is actively used.
```
Comment (wouterj) [inline README.md:4]:
```suggestion
The CMF project focusses on the [Routing component](https://github.com/symfony-cmf/routing) and [RoutingBundle](https://github.com/symfony-cmf/routing-bundle), which are still in active use by other projects.
```

**Review by wouterj — APPROVED** [review]
Would it make sense to let them know to get in contact (i.e. `#symfony_cmf` on Slack) if they want to take over ownership?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# This repository is no longer maintained
+
+Due to lack of interest, we had to decide to discontinue this repository. 
+The CMF project focusses on the [Routing component](https://github.com/symfony-cmf/routing) and [RoutingBundle](https://github.com/symfony-cmf/routing-bundle), which are still in active use by other projects.
+
+This repository will no longer be upgraded and marked as abandoned, but will be kept available for legacy projects or if somebody wants to experiment with the CMF.
```
Comment (dbu) [inline README.md:6]:
```suggestion
This repository will no longer be upgraded and marked as abandoned, but will be kept available for legacy projects or if somebody wants to experiment with the CMF.

You can contact us in the #symfony_cmf channel of the [Symfony devs slack](https://symfony.com/slack).
```

**Comment by dbu** [comment]
> Would it make sense to let them know to get in contact (i.e. #symfony_cmf on Slack) if they want to take over ownership?

i am somewhat worried of scams trying to get into the cmf - or even just people contacting us to reopen the repositories and then not investing the time to move things forward again... i added a link to the slack channel, i'd hope people who want to resurrect the cmf would start talking to us and we can figure out how to proceed.

**Pull request merged**